### PR TITLE
Add context arranger function

### PR DESCRIPTION
**Pull request by YuvHayun**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by YuvHayun** [comment]
@ShahafBenYakir need force merge due to Xsoar NG issues.

File: Packs/Base/Scripts/CommonServerPython/CommonServerPython.py [hunk 1]
```diff
@@ -7777,6 +7777,37 @@ def string_to_context_key(string):
         raise Exception('The key is not a string: {}'.format(string))
 
 
+def response_to_context(reponse_obj):
+    """Recursively creates a data dictionary where all key starts with capital letters.
```
Comment (ShahafBenYakir) [inline Packs/Base/Scripts/CommonServerPython/CommonServerPython.py:7781]: It not only capitalizes the string it also splits words separated by underscore.
Let's add examples of input and output as you have in the `string_to_context_key`

File: Packs/Base/Scripts/CommonServerPython/CommonServerPython.py [hunk 2]
```diff
@@ -7777,6 +7777,37 @@ def string_to_context_key(string):
...
+
+    :type reponse_obj: ``Any``
+    :param reponse_obj: The response object to update.
+
+    :return: A response with all keys (if there're any) starts with a capital letter.
+    :rtype: ``Any``
+    """
+    predefined_keys = {"id": "ID", "uuid": "UUID"}
```
Comment (ShahafBenYakir) [inline Packs/Base/Scripts/CommonServerPython/CommonServerPython.py:7789]: The developer using `response_to_context` should be able to send additional keys

Comment (YuvHayun) [reply Packs/Base/Scripts/CommonServerPython/CommonServerPython.py:7789]: Great idea, adding the option with relevant tests

Comment (YuvHayun) [reply Packs/Base/Scripts/CommonServerPython/CommonServerPython.py:7789]: Great idea, adding the option with relevant tests.

File: Packs/Base/Scripts/CommonServerPython/CommonServerPython.py [hunk 3]
```diff
@@ -7777,6 +7777,45 @@ def string_to_context_key(string):
...
+    """
+    Recursively creates a data dictionary where all key starts with capital letters.
+    If a key include underscores,  removes underscores, capitalize every word.
+    Example: "one_two" to "OneTwo
+
+    :type reponse_obj: ``Any``
+    :param reponse_obj: The response object to update.
+    :type reponse_obj: ``dict``
+    :user_predefiend_keys: An optional argument, 
+    a dict with predefined keys where the key is the key in the response and value is the key we want to turn the key into.
```
Comment (ShahafBenYakir) [inline Packs/Base/Scripts/CommonServerPython/CommonServerPython.py:7790]:
```suggestion
    :param user_predefiend_keys: a dictionary where each key and value represent a key in the reponse_obj and the value to turn the key into respectively.
```

File: Packs/Base/Scripts/CommonServerPython/CommonServerPython.py [hunk 4]
```diff
@@ -7777,6 +7777,45 @@ def string_to_context_key(string):
         raise Exception('The key is not a string: {}'.format(string))
 
 
+def response_to_context(reponse_obj, user_predefiend_keys):
```
Comment (ShahafBenYakir) [inline Packs/Base/Scripts/CommonServerPython/CommonServerPython.py:7780]: user_predefiend_keys is a positional argument and hence developers will need to do `response_to_context(my_dict, None)` in order to net get an error. 
let's add default value `None` to it
```suggestion
def response_to_context(reponse_obj, user_predefiend_keys=None):
```

File: Packs/Base/Scripts/CommonServerPython/CommonServerPython.py (hunk 3 above)
Comment (ShahafBenYakir) [reply Packs/Base/Scripts/CommonServerPython/CommonServerPython.py:7790]: Also let's add a test to this addition

Comment (YuvHayun) [reply Packs/Base/Scripts/CommonServerPython/CommonServerPython.py:7790]: I added 2 already.

**Review by ShahafBenYakir — APPROVED** [review]

**Pull request merged**